python/ray/tests/test_actor_failures.py

```diff
@@ -703,7 +733,10 @@ def load_checkpoint(self, actor_id, checkpoints):
     assert ray.get(actor.was_resumed_from_checkpoint.remote()) is False
 
     # Check that the checkpoint error was pushed to the driver.
-    wait_for_errors(ray_constants.CHECKPOINT_PUSH_ERROR, 1)
+    errors = wait_for_message(p, 2)
```

Comment:
Why do we wait for 2 msgs?

Reply:
1 or 2 is same, it just timeout because there is no error message. I will change it to 1